package.json

```diff
@@ -2,7 +2,7 @@
   "name": "mongoman",
   "author": "John Hofrichter <https://github.com/johnhof>",
   "description": "MongoDB management utility",
-  "version": "0.0.4",
+  "version": "1.0.0",
```

Comment:
@johnhof I see that you are updating rev, which makes sense, but shouldn't this actually be 1.0.1? Actually this may be big enough of a change to go to 1.1.0. I'll leave that to you to decide.